src/scripts/farming/mutation/mutationTypes/EnigmaMutation.ts

```diff
@@ -58,7 +58,22 @@ class EnigmaMutation extends GrowMutation {
      */
     static getReqs(): BerryType[] {
         SeededRand.seed(+App.game.discord.ID());
-        const berryTypes = Farming.getGeneration(2).concat(Farming.getGeneration(3));
+        // Getting possible Berries
+        // Only Gen 3 and 4 Berries
+        let berryTypes = Farming.getGeneration(2).concat(Farming.getGeneration(3));
+        // Remove parasite Berries
```

Comment:
Can we make this a little more descriptive of why these certain berries are being removed.

Reply:
Updated.
Though theoretically with my Babiri changes this isn't that necessary, but it would be arbitrarily harder for some players, so it's probably better to remove these from the pool.